MISC/ShootSingleBulletBetweenCoordsIgnoreEntityNew.md

```diff
@@ -6,32 +6,31 @@ aliases: ["0xBFE5756E7407064A","_SHOOT_SINGLE_BULLET_BETWEEN_COORDS_WITH_EXTRA_P
 
 ```c
 // 0xBFE5756E7407064A 0xCCDC33CC
-void SHOOT_SINGLE_BULLET_BETWEEN_COORDS_IGNORE_ENTITY_NEW(float x1, float y1, float z1, float x2, float y2, float z2, int damage, BOOL p7, Hash weaponHash, Ped ownerPed, BOOL isAudible, BOOL isInvisible, float speed, Entity entity, BOOL p14, BOOL p15, BOOL p16, BOOL p17);
+void SHOOT_SINGLE_BULLET_BETWEEN_COORDS_IGNORE_ENTITY_NEW(float x1, float y1, float z1, float x2, float y2, float z2, int damage, BOOL pureAccuracy, Hash weaponHash, Ped ownerPed, BOOL isAudible, BOOL isInvisible, float speed, Entity entity, BOOL p14, BOOL p15, BOOL p16, BOOL p17, Any p18, Any p19, Any p20);
```

Comment:
You can add that `p13` (entity) can be set to false to ignore entity.

`p14` -> BOOL to force to create a new projectile object.
`p15` -> BOOL if it disable or not the player cover start adjustement.
`p16` -> the target entity index.
`p17` -> BOOL not sure what it does.
`p18` -> BOOL to freeze or not the projectile waiting to hit collision. False by default.
`p19` ->BOOL collision to ignore collision with entity. False by default,
`p20` -> BOOL collision to ignore collision until the projectile leaves the BoundingBox of the given IgnoreEntity

Reply:
p14 = bool bForceCreateNewProjectileObject
p15 = bool bDisablePlayerCoverStartAdjustment
p16 = Entity targetEntity
p17 = bool bDoDeadCheck
p18 = bool bFreezeProjectileWaitingOnCollision
p19 = bool bSetIgnoreCollisionEntity
p20 = bool bIgnoreCollisionResetNoBB

would be the correct types and names